src/buckets/actions/thunks.ts

```diff
@@ -305,6 +313,37 @@ export const deleteBucketLabel = (bucketID: string, label: Label) => async (
   }
 }
 
+export const addSchemasToBucket = (
+  bucketID: string,
+  orgID: string,
+  bucketName: string,
+  schema: typeof MeasurementSchemaCreateRequest
+) => async (dispatch: Dispatch<Action>) => {
+  const params = {
+    bucketID,
+    data: schema,
+    query: {orgID},
+  }
+  try {
+    const resp = await postBucketsSchemaMeasurement(params)
+    if (resp.status !== 201) {
+      const msg = resp?.data?.message
+      console.error('error adding measurement schema:', resp)
+      throw new Error(msg)
+    }
+
+    dispatch(
+      notify(measurementSchemaAdditionSuccessful(bucketName, schema.name))
+    )
+  } catch (error) {
+    console.error(error)
+    const message = getErrorMessage(error)
+    dispatch(
+      notify(measurementSchemaAdditionFailed(bucketName, schema.name, message))
+    )
```

Comment:
hmm....i agree that the error handling could be improved.  i copied the pattern from the updateBucket method.

to be consistent; changing them both would be necessary.

i vote to put this as a tech debt enhancement and leave it as it is (since this pattern is already handled by the code.)

Reply:
>i vote to put this as a tech debt enhancement and leave it as it is (since this pattern is already handled by the code.)

fair enough. Would you mind writing up a ticket for that please?

Reply:
done; https://app.zenhub.com/workspaces/applicationdumplings-60a80d872533670017a6b107/issues/influxdata/ui/2799